package.json

```diff
@@ -234,6 +234,11 @@
 				"title": "Ask Sourcery",
 				"category": "Sourcery"
 			},
+			{
+				"command": "sourcery.chat.toggleCodeLens",
+				"title": "Toggle Sourcery Code Lens",
+				"category": "Sourcery"
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Add a when clause here too?

Reply:
I don't think you add `when` clauses when defining the commands?